### PR TITLE
feat: update sig-docs-es member list

### PR DESCRIPTION
**Pull request by raelga**
@alexbrand is stepping down from the sig-docs team as owner. This PRs removes him from the owner and maintainer teams, will remain part of the team sig-docs-es reviewer.

@electrocucaracha will be the new owner for the sig-docs-es team.

/cc @alexbrand

### CODE REVIEW

**Comment by raelga** [comment]
/assign @kbhawkey
(This week PR wrangler)

**Comment by kbhawkey** [comment]
/lgtm
@raelga , Are you required to have 2 owners or can this PR move forward as is?

**Comment by raelga** [comment]
@kbhawkey  I think we can move forward, I will open a new PR with the new member this week!

Thanks :)

**Comment by raelga** [comment]
@kbarnard10 let me know if there anything else that I need to add to this PR. :)

**Comment by sftim** [comment]
@raelga is there anyone who can vouch for this PR? Perhaps @alexbrand - or another contributor to the Spanish localization?

**Comment by sftim** [comment]
BTW I believe that the usual process is to update https://github.com/kubernetes/org/blob/main/config/kubernetes/sig-docs/teams.yaml first and then mirror those changes here.

**Comment by raelga** [comment]
I spoke with @alexbrand some time ago and agreed on stepping down from the role. The clean up is just to avoid PRs getting assigned to him.

On today's weekly meeting we will discuss the new approver for the Spanish team to replace @alexbrand and we will create a PR with the updated list.

Anyhow, @electrocucaracha can probably vouch this PR.

**Review by electrocucaracha — APPROVED** [review]

**Comment by raelga** [comment]
@sftim @electrocucaracha will be the new approver for the Spanish team. We will create a PR for that request once this one is merged.

**Comment by raelga** [comment]
@sftim @kbarnard10 Let me know if you need anything else to get this PR merged, so we can open a new PR for the new approver without having merge conflicts.

**Comment by raelga** [comment]
@sftim @kbarnard10 To avoid the second PR, I'm including the new approver for the sig-docs-es team (@electrocucaracha) in this PR.

/retitle feat: update sig-docs-es member list
/auto-cc

File: config/kubernetes/sig-docs/teams.yaml
Comment (kbhawkey) [inline config/kubernetes/sig-docs/teams.yaml:316]: 👋 @raelga .
Do you want to add `@electrocucaracha` to this section?

**Comment by kbhawkey** [comment]
The changes look okay.
/lgtm

File: config/kubernetes/sig-docs/teams.yaml
Comment (raelga) [reply config/kubernetes/sig-docs/teams.yaml:316]: Thanks @kbhawkey for pointing that out! 

At this stage, we are not using any of the capabilities of the `website-mantainers` role as we don't have any branching strategy. But up to you @electrocucaracha!

**Pull request merged**
